index.html

```diff
@@ -2,69 +2,120 @@
 <html>
 <head>
 	<title>HTML forms</title>
+	<link href="styles.css" rel="stylesheet" type="text/css">
+	<link href="https://fonts.googleapis.com/css?family=Nunito&display=swap" rel="stylesheet">
+	<meta charset="utf-8">
 </head>
 <body>
 	<main>
```

Comment:
`AWESOME` 

Very good, using `main`, this shows the dominant content of the `<body>` of a document. More on `<main>` [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) :star2: :star: